src/transports/ws/bws.c

```diff
@@ -150,6 +153,7 @@ int nn_bws_create (struct nn_ep *ep)
 
     rc = nn_bws_listen (self);
     if (rc != 0) {
+        nn_free(self);
```

Comment:
This is almost surely incorrect.  The problem here is that we have the state machine setup and running, and a usock set up to use it as well.

Simply freeing is inadequate here, and leads to a segfault.